src/data/index.ts

```diff
@@ -23,11 +23,125 @@ const BACKEND_GET = "https://json.excalidraw.com/api/v1/";
 const BACKEND_V2_POST = "https://json.excalidraw.com/api/v2/post/";
 const BACKEND_V2_GET = "https://json.excalidraw.com/api/v2/";
 
+export const SOCKET_SERVER = "https://excalidraw-socket.herokuapp.com";
+
 // TODO: Defined globally, since file handles aren't yet serializable.
 // Once `FileSystemFileHandle` can be serialized, make this
 // part of `AppState`.
 (window as any).handle = null;
 
+function byteToHex(byte: number): string {
+  return `0${byte.toString(16)}`.slice(-2);
+}
+
+async function generateRandomID() {
+  var arr = new Uint8Array(20);
```

Comment:
That's 8^20 ~= 10^19 possibilities. This is really big. Since all the state is transient and you need the private key to be able to decrypt, this is not a security feature. We just need it to be big enough to avoid conflicts.

Uint8Array(10) is ~1 billion. This should be enough.

Reply:
Makes sense!